demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -220,6 +221,113 @@ def generate_setup_section(yaml_data: dict):
     return section
 
 
+# Incident Mirroring
+
+def trigger_generate_mirroring_section(yml_data: dict):
+    """
+
+    Args:
+        yml_data: yml data of the integration.
+
+    Returns:
+        true if mirroring section should be generated.
+
+    """
+    script_data = yml_data.get('script', {})
+    sync_in = script_data.get('isremotesyncout', False)
+    sync_out = script_data.get('isremotesyncin', False)
+    return sync_out and sync_in
+
+
+def is_configuration_exists(yml_data: dict, names: list):
+    """
+    Args:
+        yml_data: yml data of the integration
+        names: list of configuration params to search for
+
+    Returns:
+        the first cong if name exists in the yml_data, else none.
+
+    """
+
+    for conf in yml_data.get('configuration', []):
+        if conf.get('name', '') in names:
+            return conf
+    return None
+
+
+def generate_mirroring_section(yaml_data: dict):
+    """
+
+    Args:
+        yaml_data: dict representing the yml file of the integration.
+
+    Returns: markdown section of Incident Mirroring.
+
+    """
+    integration_name = format(yaml_data['display'])
+    directions = {
+        'None': 'Turns off incident mirroring.',
+        'Incoming': f'Any changes in {integration_name} incidents (%%mirroring incoming fields%%) will be reflected in XSOAR incidents.',
+        'Outgoing': f'Any changes in XSOAR incidents will be reflected in {integration_name} incidents (%%mirroring outgoing fields%%).',
+        'Incoming And Outgoing': f'Changes in XSOAR incidents and {integration_name} incidents will be reflected in both directions.'
```

Comment:
```suggestion
        'Incoming': f'Any changes in {integration_name} %%incident type%%% (%%mirroring incoming fields%%) will be reflected in XSOAR incidents.',
        'Outgoing': f'Any changes in XSOAR incidents will be reflected in {integration_name} %%incident type%%% (%%mirroring outgoing fields%%).',
        'Incoming And Outgoing': f'Changes in XSOAR incidents and {integration_name} %%incident type%%% will be reflected in both directions.'
```
I don't know if marking using `%%` is the way we do it on other places, this should be checked.